### ISSUE TITLE
Expected Broccoli node, got undefined with version 3.0.0

### ISSUE THREAD

**Issue by josemarluedke** [martndemus/ember-font-awesome#120]
Just tried to get version 3.0.0 of `ember-font-awesome` with Ember CLI 2.12.0-beta.1 and I get the following error: 

```
BroccoliMergeTrees: Expected Broccoli node, got undefined for inputNodes[0]
TypeError: BroccoliMergeTrees: Expected Broccoli node, got undefined for inputNodes[0]
    at BroccoliMergeTrees.Plugin (/Users/josemarluedke/Projects/main/node_modules/broccoli-plugin/index.js:23:13)
    at new BroccoliMergeTrees (/Users/josemarluedke/Projects/main/node_modules/broccoli-merge-trees/index.js:42:10)
    at BroccoliMergeTrees (/Users/josemarluedke/Projects/main/node_modules/broccoli-merge-trees/index.js:36:53)
    at Class.treeForVendor (/Users/josemarluedke/Projects/main/node_modules/ember-font-awesome/index.js:34:12)
    at Class._treeFor (/Users/josemarluedke/Projects/main/node_modules/ember-cli/lib/models/addon.js:517:33)
    at Class.treeFor (/Users/josemarluedke/Projects/main/node_modules/ember-cli/lib/models/addon.js:477:21)
    at project.addons.map.addon (/Users/josemarluedke/Projects/main/node_modules/ember-cli/lib/broccoli/ember-app.js:525:20)
    at Array.map (native)
    at EmberApp.addonTreesFor (/Users/josemarluedke/Projects/main/node_modules/ember-cli/lib/broccoli/ember-app.js:523:30)
    at EmberApp._processedVendorTree (/Users/josemarluedke/Projects/main/node_modules/ember-cli/lib/broccoli/ember-app.js:960:29)
```

It seems that `treeForVendor` receives `undefined` and we try to merge the trees with that.

**Comment by martndemus** [comment]
Fixed in 3.0.1!

**Comment by josemarluedke** [comment]
Thank you @martndemus!
